portmapper/mapper.go

```diff
@@ -174,7 +152,7 @@ func (pm *PortMapper) MapRange(container net.Addr, hostIP net.IP, hostPortStart,
 
 	containerIP, containerPort := getIPAndPort(m.container)
 	if hostIP.To4() != nil {
-		if err := pm.forward(iptables.Append, m.proto, hostIP, allocatedHostPort, containerIP.String(), containerPort); err != nil {
+		if err := pm.forwardIptableAppend(m.proto, hostIP, allocatedHostPort, containerIP.String(), containerPort); err != nil {
```

Comment:
Just a minor comment . Can you pls replace the function name to `AppendForwardingTableEntry` ? 
Similarly change the other funciton to `DeleteForwardingTableEntry` ?

With this PR, since we are making mapper.go is OS agnostic, it is better to not have `iptables` directly in the function name.